react/Popover.tsx

```diff
@@ -8,13 +8,17 @@ import TrapFocus from './components/TrapFocus'
 import Fade from './components/Animations/Fade'
 import OutsideClickHandler from './components/OutsideClickHandler'
 import { usePopoverState, usePopoverDispatch } from './PopoverContext'
+import useEventCallback from './modules/useEventCallback'
+import handleContainerStyle, { RestoreFn } from './modules/handleContainerStyle'
 
 type TransitionComponentType = 'fade'
 
 interface Props {
-  children: React.ReactNode
-  placement: Placement
   role?: string
+  placement: Placement
+  children: React.ReactNode
+  backdrop?: 'clickable' | 'none'
+  onScroll?: 'block' | 'close' | 'none'
```

Comment:
Not sure about those names yet
@lbebber halp

Reply:
q tal `scrollBehavior: lock-scroll | close-popover | regular`

Reply:
ou talvez `scrollBehavior: lock-page-scroll | close-popover-on-scroll | default`
um pouquinho mais verboso mas mais claro eu acho

Reply:
Esse segundo valor eu acho que ficaria melhor `close-on-scroll', o que tu acha?

Reply:
Pode ser